src/api.js

```diff
@@ -446,6 +453,21 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
         return rowObject;
     };
 
+    /** Get the SQL string used in preparing this statement.
+     @return {string} The SQL string
+     */
+    Statement.prototype["getSQL"] = function getSQL() {
+        return sqlite3_sql(this.stmt);
+    };
+
+    /** Get the SQLite's normalized version of the SQL string used in
```

Comment:
We need to document what *normalized* means.

Reply:
I phrased it the way I did because SQLite doesn't actually define what normalized means.  What they say is: "The semantics used to normalize a SQL statement are unspecified and subject to change. At a minimum, literal values will be replaced with suitable placeholders."  In practice in the current version, e.g., comments and unnecessary spaces are removed and keywords are capitalized (which gives me what I need), but no guarantees on what it will mean in future.

Reply:
Then maybe we can add an `@example` and a link to SQLite's documentation in the jsdoc.